### ISSUE TITLE
While trying to install any missing Dotnet Core 2.1 Dependencies, I get a NSS conflict error.

### ISSUE THREAD

**Issue by lukegawel** [microsoft/azure-pipelines-agent#2469]
While trying to run the agent register script for linux, it promts to install any dotnet core dependencies with the installdependencies.sh script. Afterward running that script, I get the error below:

![Capture](https://user-images.githubusercontent.com/54082112/65548825-400fc680-deea-11e9-83f0-db816e608c57.PNG)

Error: nss-pem conflicts with nss-3.21.3-2.el7_3.x86_64
Error: Package: nss-pem-1.0.3-7.el7.x86_64 (base)
           Requires: nss(x86-64) >= 3.36.0
           Installed: nss-3.21.3-2.el7_3.x86_64 (@updates)
               nss(x86-64) = 3.21.3-2.el7_3
Error: Package: libcurl-7.29.0-54.el7.x86_64 (base)
           Requires: libnss3.so(NSS_3.34)(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
'yum' failed with exit code '0'
Can't install dotnet core dependencies.
You can manually install all required dependencies based on following documentation

I tried both --skip-borken and rpm- and neither helped.  The CenTos Linux release is 7.3.1611, but the dev box is the same version and was able to have the agent installed with no problem.

**Comment by jtpetty** [comment]
@lukegawel - I apologize for the late reply, but were you able to get around your problem?

**Comment by jtpetty** [comment]
Closing due to inactivity. If the problem still exists, please feel free to open a new issue.
